.gitignore

```diff
@@ -1 +1,5 @@
 build/*
```

Comment:
I would exclude `build` folder itself not just the children ones (no /*)

Reply:
done in d1b2d31